CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* Adds new field `pool.allocation.fixed_values` to `v1/config/ResourcePools`
```

Comment:
Can you also call out the new log scale commands (action/alert)

Reply:
Done